docs/samples/browser-plugin-meetings/index.html

```diff
@@ -1,508 +1,594 @@
+            <h5 id="meeting-info-subtitle" style="margin-top: 0;">25783912061@cisco.webex.com
+              (92e04d6f-8c70-4b9b-886f-9e11c13454f3)</h5>
+          </div>
+          <div>
+            <button id="meetings-leave" type="button" onclick="leaveMeeting()">Leave</button>
+          </div>
+        </div>
+        <div class="section-content collapsed">
 
+        </div>
+      </section>
 
-        <!-- ############################## ############################## ############################## -->
-        <!-- meeting-controls -->
-        <section id="meetingControls">
-          <h2>Meeting Controls</h2>
-          <p>The following manipulate the <code>meeting</code> object.</p>
-          <ol>
-            <li><a href="#meetingControlsMuteAudio">Mute / Unmute Audio</a></li>
-            <li><a href="#meetingControlsMuteVideo">Mute / Unmute Video</a></li>
-            <li><a href="#meetingControlsScreenshare">Screenshare</a></li>
-            <li><a href="#meetingControlsRecording">Recording</a></li>
-            <li><a href="#meetingControlsLocking">Lock / Unlock</a></li>
-            <li><a href="#meetingControlsTranscription">Transcription</a></li>
-            <li><a href="#meetingControlsTones">Tones</a></li>
-          </ol>
-          <br>
-
-          <!-- meeting-controls / mute-unmute audio -->
-          <fieldset id="meetingControlsMuteAudio">
-            <legend>Mute/Unmute Audio</legend>
+      <!-- meeting streams -->
+      <section class="section-box">
+        <h2 class="collapsible">Meeting Streams</h2>
+        <div class="section-content">
+          <div class="video-section">
             <div>
-                <p class="note">NOTE: Must have an active meeting object.</p>
-                <div class="u-mb">
-                  <button id="ts-toggle-audio" type="button" onclick="toggleSendAudio()">Toggle Audio</button>
-                  <div class="btn-group">
-                      <button type="button" class="btn-code" onclick="meeting.muteAudio()">meeting.muteAudio()</button>
-                      <button type="button" class="btn-code" onclick="meeting.unmuteAudio()">meeting.unmuteAudio()</button>
+              <div class="flex" style="gap: 0.5rem;">
+                <fieldset>
+                  <legend>Local Video</legend>
+                  <video id="local-video" muted="true" autoplay playsinline style="height: 200px;"></video>
+                </fieldset>
+
+                <fieldset style="display: flex; align-items: center;">
+                  <legend>Local Audio</legend>
+                  <audio id="local-audio" controls autoplay></audio>
+                </fieldset>
+              </div>
+              <div class="flex">
+                <!--mute-unmute audio -->
+                <div>
+                  <div class="u-mb">
+                    <button id="ts-toggle-audio" type="button" onclick="toggleSendAudio()">Toggle Audio</button>
```

Comment:
I feel, that instead of saying toggle Audio or Toggle video, Simply we can say Mute Video or Unmute Video or Audio. Or best will be to use the Icons for the best UI.